api/src/resources/auth/auth.service.ts

```diff
@@ -63,6 +62,46 @@ class AuthService {
         });
     }
   }
+
+  public async login(req: Request, res: Response): Promise<Response | void> {
+    const userRepository = AppDataSource.getRepository(User);
+    try {
+      const { email, password } = req.body;
+
+      // Validation basique
+      if (!email || !password) {
+        return res.status(400).json({ message: "All fields are required." });
+      }
+
+      // Vérifier si l'utilisateur existe
+      const user = await userRepository.findOne({ where: { email } });
+      if (!user) {
+        return res.status(401).json({ message: "Invalid email" });
```

Comment:
Ce type de réponse est déconseillé, ça rend ton site vulnérable aux attaques par énumération.
En gros ne dis pas que l'adresse email est invalide ou le mot de pass est invalide, quelque soit le cas, dis que les infos de connexion sont incorrectes.
https://medium.com/@leilaalvess/what-is-user-enumeration-c7d5a56eeba0